build/walk_test.go

```diff
@@ -0,0 +1,43 @@
+package build
+
+import "testing"
+
+func nodeToString(e Expr) string {
+	if bin, ok := e.(*BinaryExpr); ok {
+		return bin.Op
+	} else if lit, ok := e.(*LiteralExpr); ok {
+		return lit.Token
+	} else {
```

Comment:
can drop else, return in previous branch

Reply:
done